packages/shared/src/lib/core/layer-2/actions/fetchL2BalanceForAccount.ts

```diff
@@ -116,7 +113,7 @@ async function getLayer2Erc20BalancesForAddress(
                 continue
             }
             const rawBalance = await contract.methods.balanceOf(evmAddress).call()
-            erc20TokenBalances.push({ balance: Math.floor(Number(rawBalance)), tokenId: erc20Address })
+            erc20TokenBalances.push({ balance: BigInt(rawBalance), tokenId: erc20Address })
```

Comment:
is the new convertor function safer here for type any?

Reply:
Same response as above.